src/presence_analyzer/tests.py

```diff
@@ -85,6 +147,41 @@ def test_get_data(self):
             data[10][sample_date]['start'],
             datetime.time(9, 39, 5)
         )
+        self.assertTrue(mock_log.debug.called)
+
+    def test_mean(self):
+        """
+        Test mean() function.
+        """
+        mean = utils.mean
```

Comment:
no need for such reassignment, can be removed here and in subsequent tests as well.